src/components/configuration/Configuration.ts

```diff
@@ -96,7 +96,7 @@ export class Configuration {
             undefined
         );
 
-        configuration.updatePathToRlsExecutableSpecifiedByUser();
+        configuration.updateUserRlsPath();
```

Comment:
17 characters are long enough. 30 is not acceptable. Long lines are not really readable.

You're confusing function names with documentation comments.